code/espurna/relay.ino

```diff
@@ -732,6 +732,10 @@ void _relayWebSocketSendRelays(JsonObject& root) {
     JsonArray& pulse = relays.createNestedArray("pulse");
     JsonArray& pulse_time = relays.createNestedArray("pulse_time");
 
+    #if SCHEDULER_SUPPORT
+        JsonArray& relayLastschedule = relays.createNestedArray("relayLastschedule");
+    #endif
```

Comment:
IMO this should be something like `sch_last`
Also note that neighbour PR re-does things in vue, and we are using short things here

Reply:
Changed to sch_last, also fixed it in the JS...